docs/src/aleph_bft_api.md

```diff
@@ -67,43 +67,32 @@ pub trait KeyBox: Index + Clone + Send {
 
 A typical implementation of KeyBox would be a collection of `N` public keys, an index `i` and a single private key corresponding to the public key number `i`. The meaning of `sign` is then to produce a signature using the given private key, and `verify(msg, s, j)` is to verify whether the signature `s` under the message `msg` is correct with respect to the public key of the `j`th node.
 
-#### 3.1.3 IntoBackup, IntoReader.
+#### 3.1.3 Read, Write
```

Comment:
I would title this to make it more explicit that it's about intra-session recovery, perhaps "Read, Write – recovering mid session crashes".

I would also add a paragraph that this is optional if you don't want to recover crashes mid session, and if your session handling ensures AlephBFT will not run in the same session twice you can pass noop implementations here.

Preferably there should also be better documentation about turning initial unit collection off in this case, but that probably shouldn't be in this PR. >.<

Reply:
I probably wasn't clear – I think the paragraph about this being optional should be in this PR.